ui/apps/platform/src/Containers/Clusters/ClusterEditForm.tsx

```diff
@@ -10,10 +10,8 @@ import ClusterSummary from './Components/ClusterSummary';
 import StaticConfigurationSection from './StaticConfigurationSection';
 import DynamicConfigurationSection from './DynamicConfigurationSection';
 import ClusterLabelsTable from './ClusterLabelsTable';
-import { CentralEnv } from './clusterTypes'; // augmented with successfullyFetched
```

Comment:
What is the purpose of the deletions in this file?

Reply:
This is the result of the tightly enforced linting rules in the UI code, which disallows unused code.